frontend/www/js/omegaup/components/submissions/Feedback.vue

```diff
@@ -68,14 +73,22 @@ import user_Username from '../user/Username.vue';
 export default class SubmissionFeedback extends Vue {
   @Prop() guid!: string;
   @Prop({ default: false }) isAdmin!: boolean;
-  @Prop({ default: null }) feedbackOptions!: null | types.SubmissionFeedback;
+  @Prop({ default: () => [] }) feedbackOptions!: types.SubmissionFeedback[];
 
   T = T;
   ui = ui;
   time = time;
 
   showFeedbackForm = false;
-  feedback = this.feedbackOptions?.feedback ?? null;
+  feedback = this.generalFeedback?.feedback ?? null;
+
+  get generalFeedback(): null | types.SubmissionFeedback {
+    if (!this.feedbackOptions.length) return null;
+    const [feedback] = this.feedbackOptions.filter(
+      (feedback) => feedback.range_bytes_start == 0,
+    );
+    return feedback;
```

Comment:
```suggestion
    return feedback ?? null;
```

`feedback` puede ser `undefined`, no `null`.

Reply:
Se arregla en el PR #6862